src/data/reducers/filters.test.js

```diff
@@ -0,0 +1,172 @@
+    const expected = {
+      ...testingState,
+      assignmentType: expectedFilterType,
+    };
+    expect(filter(testingState, actions.update.assignmentType({
+      filterType: expectedFilterType,
+    }))).toEqual(expected);
+  });
+
+  it('filter on not existed assignment type', () => {
+    const notExistFilter = 'not exist filter';
+    const expected = {
+      ...testingState,
+      assignmentType: notExistFilter,
+      assignment: '',
+    };
+    expect(filter(testingState, actions.update.assignmentType({
+      filterType: notExistFilter,
+    }))).toEqual(expected);
+  });
+
+  it('initialize', () => {
+    const payload = {
+      assignment: expectedAssignment,
+      assignmentType: expectedFilterType,
+      track: 'verified',
+      cohort: 5,
+      assignmentGradeMin: 50,
+      assignmentGradeMax: 100,
+      courseGradeMin: 50,
+      courseGradeMax: 100,
+      includeCourseRoleMembers: true,
+    };
+    const expected = {
+      ...initialState,
+      ...payload,
+      assignment: {
+        id: payload.assignment,
+      },
+    };
+    expect(filter(undefined, actions.initialize(payload))).toEqual(expected);
+  });
+
+  it('reset', () => {
+    const payload = Object.keys(initialFilters);
+    expect(filter(testingState, actions.reset(payload))).toEqual(initialFilters);
+  });
+
+  it('update assignment', () => {
+    const expected = {
+      ...initialState,
+      assignment: expectedAssignment,
+    };
+    expect(
+      filter(initialState, actions.update.assignment(expectedAssignment)),
+    ).toEqual(expected);
+  });
+
+  it('update assignment limit', () => {
+    const expectedMinGrade = 50;
+    const expectedMaxGrade = 100;
+    const expected = {
+      ...initialState,
+      assignmentGradeMin: expectedMinGrade,
+      assignmentGradeMax: expectedMaxGrade,
+    };
+    expect(
+      filter(initialState, actions.update.assignmentLimits({
+        minGrade: expectedMinGrade,
+        maxGrade: expectedMaxGrade,
+      })),
+    ).toEqual(expected);
+  });
+
+  it('update grade limit', () => {
+    const expectedCourseMinGrade = 50;
+    const expectedCourseMaxGrade = 100;
+    const expected = {
+      ...initialState,
+      courseGradeMin: expectedCourseMinGrade,
+      courseGradeMax: expectedCourseMaxGrade,
+    };
+    expect(
+      filter(initialState, actions.update.courseGradeLimits({
+        courseGradeMin: expectedCourseMinGrade,
+        courseGradeMax: expectedCourseMaxGrade,
+      })),
+    ).toEqual(expected);
+  });
+
+  it('update include course role members', () => {
+    const expectedIncludeCourseRoleMembers = true;
+    const expected = {
+      ...initialState,
+      includeCourseRoleMembers: expectedIncludeCourseRoleMembers,
+    };
+    expect(
+      filter(initialState,
+        actions.update.includeCourseRoleMembers(expectedIncludeCourseRoleMembers)),
+    ).toEqual(expected);
+  });
+});
+
+describe('grade reducer group', () => {
```

Comment:
this distinction should be a describe block in the above, and has nothing to do with a grade reducer, but rather the handling of grades actions.